tools/sslsniff.py

```diff
@@ -23,17 +23,25 @@
 examples = """examples:
     ./sslsniff              # sniff OpenSSL and GnuTLS functions
     ./sslsniff -p 181       # sniff PID 181 only
+    ./sslsniff -u 1000      # sniff only UID 1000
```

Comment:
Since you are touching this file, could you also update line 7:
 USAGE: sslsniff.py [-h] [-p PID] [-c COMM] [-o] [-g] [-d]
clearly, we has missed some flags there. You don't need to add --ebpf which is mostly hidden from tool users.